optee-client.spec

```diff
@@ -119,6 +119,8 @@ systemctl daemon-reload >/dev/null 2>&1 || :
 %{_sbindir}/tee-supplicant
 %{_sysconfdir}/modprobe.d/tpm_ftpm_tee.conf
 %{_unitdir}/tee-supplicant.service
+%{_libdir}/pkgconfig/teeacl.pc
+%{_libdir}/pkgconfig/teec.pc
```

Comment:
The .spec file is for RPM.
Have we verified it on Ubuntu (.deb) as well, such as run "debuild -us -uc" inside the optee-client directory on the DPU locally?

Reply:
Yes, verified. I updated the commit above to show how Debian was tested and verified.